### PR TITLE
Rename group name from format/audit to prebuild/postbuild

### PR DESCRIPTION
**Pull request by leemgs**
Rename group name from format/audit to prebuild/postbuild

This commit is to replace the existing group name to improve
code readability and maintenance.

* Before: format group, audit group
* After:  prebuild groupd, postbuild group

**Changelog**
* V2
  - Added changelog of debian package
  - Updated README.md
  - Updated the configuration file for improving maintenance

* V1
  - Replaced format/audit with prebuild/postbuild for readability
  - Fixed typos

Signed-off-by: Geunsik Lim <geunsik.lim@samsung.com>

### CODE REVIEW

**Comment by leemgs** [comment]
**Changelog**
* V2
  - Added changelog of debian package
  - Updated README.md
  - Updated the configuration file for improving maintenance

* V1
  - Replaced format/audit with prebuild/postbuild for readability
  - Fixed typos

Signed-off-by: Geunsik Lim <geunsik.lim@samsung.com>

**Comment by leemgs** [comment]
The local test is done. This PR is ready to be merged.

File: README.md [hunk 1]
```diff
@@ -1,15 +1,15 @@
-[![GitHub license](https://dmlc.github.io/img/apache2.svg)](./LICENSE)
+[![GitHub license](https://dmlc.GitHub.io/img/apache2.svg)](./LICENSE)
 
 [Build Status](http://nnsuite.mooo.com/TAOS-CI/ci/taos/) |
 [Documentation](ci/doc/doxygen-documentation.md) |
 [Contributing](ci/doc/contributing.md) |
 [Chat Room](https://gitter.im/login) |
-[Release Notes](https://github.com/nnsuite/TAOS-CI/wiki/Release-Plan)
+[Release Notes](https://GitHub.com/nnsuite/TAOS-CI/wiki/Release-Plan)
```
Comment (wooksong) [inline README.md]: Capital G? What for?

Comment (leemgs) [reply README.md]: Nice catch. Fixed.

**Pull request merged**